tests/prometheus_unit_tests/scripts/check_alert_tests.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+PROMETHEUS_CONFIG_YAML=$1
```

Comment:
@zdtsw Moved the scripts to tests/prometheus_unit_tests/scripts from [comment](https://github.com/red-hat-data-services/rhods-operator/pull/366#discussion_r1827980955)